android/build.gradle

```diff
@@ -31,6 +35,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.facebook.react:react-native:+'
-    api 'com.heapanalytics.android:heap-android-client:0.8.3'
+    implementation "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
+    api "com.heapanalytics.android:heap-android-client:0.8.3"
```

Comment:
nit: let's use single quotes here since this doesn't rely on interpolation.

Reply:
👍